grammar.js

```diff
@@ -1,17 +1,20 @@
 const PREC = {
-  union: 2,
-  intersection: 2,
-  declaration: 1,
-  type_assertion: 15,
-  as_expression: 14,
-  array_type: 13,
-  function_call: 12,
-  NEG: 9,
-  INC: 10,
+  DECLARATION: 1,
+  ACCESSIBILITY: 1,
+  UNION: 2,
+  INTERSECTION: 2,
   PLUS: 4,
   REL: 5,
   TIMES: 6,
-  non_null_assertion_op: 10
+  TYPEOF: 7,
+  NEG: 9,
+  INC: 10,
+  NON_NULL_ASSERTION_OP: 10,
+  FUNCTION_CALL: 12,
+  ARRAY_TYPE: 13,
+  MEMBER: 13,
+  AS_EXPRESSION: 14,
+  TYPE_ASSERTION: 15
```

Comment:
Added `ACCESSIBILITY`, `TYPEOF`, and `MEMBER` precedents.

The other changes here are uppercasing the precedent names for consistency, and sorted by precedent value.